cmd/start.go

```diff
@@ -156,10 +157,16 @@ var StartCmd = &cli.Command{
 			if err != nil {
 				return err
 			}
+			// TODO(forrest): eventually add option to use a file via jobqueue.NewDB()
+			pdpDB, err := jobqueue.NewInMemoryDB(cCtx.Context)
+			if err != nil {
+				return err
+			}
```

Comment:
I've stubbed this in for now, meaning the job queue isn't yet "persisted", rather it runs a sqlite database in-memory.